FlowCrypt/Controllers/Setup/SetupManuallyEnterPassPhraseViewController.swift

```diff
@@ -235,8 +235,18 @@ extension SetupManuallyEnterPassPhraseViewController {
         let keysToUpdate = Array(Set(existedKeys).intersection(fetchedKeys))
         let newKeysToAdd = Array(Set(fetchedKeys).subtracting(existedKeys))
 
-        appContext.keyStorage.addKeys(keyDetails: newKeysToAdd, passPhrase: passPhrase, source: .imported, for: email)
-        appContext.keyStorage.updateKeys(keyDetails: keysToUpdate, passPhrase: passPhrase, source: .imported, for: email)
+        appContext.encryptedStorage.addKeys(
+            keyDetails: newKeysToAdd,
+            passPhrase: passPhrase,
+            source: .imported,
+            for: email
+        )
```

Comment:
Since you switched to use encrypted storage directly, can you remove some unused methods from keyStorage?

Reply:
Since I removed `KeyStorageType` and implementation and mentioned this in comment above, I was thinking that maybe you overlooked this. Sorry, will ask you next time.

Reply:
Ah! Very good, I indeed overlooked it because the implementation is called `KeyDataStorage` ... and I was looking for `KeyStorage`. All good!